js_modules/dagit/src/SidebarSolidInfo.tsx

```diff
@@ -119,31 +122,32 @@ export default class SidebarSolidInfo extends React.Component<
             </UL>
           ) : null}
         </SectionItemContainer>
-      )
-    );
+      ));
   }
 
   renderOutputs() {
-    return this.props.solid.outputs.map((output, i: number) => (
-      <SectionItemContainer key={i}>
-        <SectionItemHeader>{output.definition.name}</SectionItemHeader>
-        <TypeWrapper>
-          <TypeWithTooltip type={output.definition.type} />
-        </TypeWrapper>
-        <Description description={output.definition.description} />
-        {output.definition.expectations.length > 0 ? (
-          <H6>Expectations</H6>
-        ) : null}
-        <UL>
```

Comment:
i dont know why caps, just leaving it as is. I only changed the one line adding the filter [1]